cylc/sphinx_ext/cylc_lang/autodocumenters.py

```diff
@@ -346,3 +367,314 @@ def run(self):
         )
 
         return [node]
+
+
+class CylcMetadataDirective(SphinxDirective):
+    """Represent a Cylc Config.
+    """
+    optional_arguments = 3
+
+    def run(self):
+        # Parse input options:
+        for key, value in zip(
+            [i.strip(':') for i in self.arguments[::2]],
+            list(self.arguments[1::2])
+        ):
+            self.options.update({key: value})
+
+        # Get global or workflow metadara
+        if 'source' in self.options:
+            config = self.load_workflow_cfg(self.options['source'])
+            metadata = self.get_workflow_metadata(
+                config, self.options['source'])
+            ret = self.workflow_to_node(metadata, config)
+        else:
+            src = self.options['global']
+            config = self.load_global_cfg(self.options['global'])
+            metadata = self.get_global_metadata(config)
+            ret = self.global_to_node(metadata, src)
+
+        node = addnodes.desc_content()
+        self.state.nested_parse(
+            StringList(ret),
+            self.content_offset,
+            node
+        )
+        return [node]
+
+    @staticmethod
+    def load_global_cfg(conf_path=None):
+        """Get Global Configuration metadata:
+
+        Args:
+            Path: Global conf path.
+
+        """
+        # Load Global Config:
+        if conf_path:
+            env = os.environ
+            sub = run(
+                ['cylc', 'config', '--json'],
+                capture_output=True,
+                env=env.update({'CYLC_SITE_CONF_PATH': conf_path})
```

Comment:
This is modifying the process environment. Make a copy if you don't want to do that!